parts/k8s/kubernetesmasterresources.t

```diff
@@ -530,7 +530,7 @@
         "autoUpgradeMinorVersion": true,
         "settings": {},
         "protectedSettings": {
-          "commandToExecute": "[concat(variables('provisionScriptParametersCommon'),' ',variables('provisionScriptParametersMaster'),' /usr/bin/nohup /bin/bash -c \"stat /opt/azure/containers/provision.complete || /bin/bash /opt/azure/containers/provision.sh >> /var/log/azure/cluster-provision.log 2>&1\"')]"
+          "commandToExecute": "[concat(variables('provisionScriptParametersCommon'),' ',variables('provisionScriptParametersMaster'), ' MASTER_INDEX=',copyIndex(variables('masterOffset')),' /usr/bin/nohup /bin/bash -c \"stat /opt/azure/containers/provision.complete || /bin/bash /opt/azure/containers/provision.sh >> /var/log/azure/cluster-provision.log 2>&1\"')]"
```

Comment:
Should MASTER_INDEX be passed in through provisionScriptParametersMaster?

Reply:
That's not possible because copyIndex() needs the copy object which is part of the resource. Putting in in provisionScriptParametersMaster was my first intention too. This is the solution @anhowe and I came up with instead.

Reply:
thanks @CecileRobertMichon , the solution above looks good